web_services/foxx/brightid/db.js

```diff
@@ -1058,6 +1085,28 @@ function setRequiredRecoveryNum(id, requiredRecoveryNum, timestamp) {
   });
 }
 
+
+function isVerifiedFor(user, app) {
```

Comment:
as asked before the `isVerifiedFor` should get `(user, verification)` as parameter